pkg/pb/inspector.proto

```diff
@@ -12,6 +12,7 @@ import "google/protobuf/timestamp.proto";
 
 package inspector;
```

Comment:
We can change inspector packages as needed and overridde the lock file, since they are not part of the communication protocol.